test/testdata/rbs/signatures_defs.rb

```diff
@@ -218,7 +218,7 @@ def foo; end # error: The method `foo` does not have a `sig`
   end
 
   def foo # error: The method `foo` does not have a `sig`
-    #: -> void # error: Unused RBS signature comment. No method definition found after it
+    #: -> void # error: Unexpected RBS assertion comment found after `method`
```

Comment:
```suggestion
    #: -> void # error: Unexpected RBS assertion comment found after `method declaration`
```

Reply:
I'll see what I can do

Reply:
Yes I think so. And this was the only assertion we had so I went with "in `method`".